eng/bomgenerator/generateAzureSDKBOM.ps1

```diff
@@ -43,6 +43,10 @@ if(! (Test-Path $InputDir)) {
   New-Item -Path $PSScriptRoot -Name "inputDir" -ItemType "directory"
 }
 
+if(! (Test-Path $OutputDir)) { 
+  New-Item -Path $PSScriptRoot -Name "outputdir" -ItemType "directory"
```

Comment:
`outputdir` or `$OutputDir` for the new directory here?

Reply:
Path has the directory name and the -Name is the outputDirName which is outputdir. So, this is intentional :)